examples/de-aws-eks/README.md

```diff
@@ -55,7 +55,7 @@ Prepare Terraform environment by setting the AWS profile and (optionally) region
 
 ```shell
 export TF_VAR_aws_profile_name="document-engine-example"
-export TF_VAR_aws_region="eu-north-1"
+export TF_VAR_aws_region="eu-north-1" # Remove or change the default in `terraform.tfvars` file if setting this variable
```

Comment:
What's wrong with Stockholm? 
Cinnamon buns are all right there.

Reply:
Nothing wrong with it but the `.tfvars` file always overrides the environment variables so if someone wants to run the example in a different region this is something to keep in mind.